src/v3/webpack.dev.config.ts

```diff
@@ -142,7 +143,7 @@ const devConfig: Configuration = mergeWithRules({
       headers,
       compress: true,
       port: DEV_SERVER_PORT,
-      proxy: [{
+      proxy: DISABLE_MOCK_SERVER ? [] : [{
```

Comment:
Don't use mock server on port 3000 for TestCafe tests. 
Tests should have own `RequestMock()`s only for endpoints they use.
Otherwise there can be implicit side effect when test uses playground's mock server if appropriate `RequestMock()` is not specified for some endpoint